Makefile

```
@@ -41,16 +41,19 @@ help: ## Show this help message
 setup: ## Initialize project: install dependencies, create .env files and pull required Docker images.
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (15).

(maxbodylength)